source/comply/audit-log.rst

```diff
@@ -1,369 +1,293 @@
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| disable_level       | bool     | Disables output of the log level display name. Default is ``false``.                    |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| disable_msg         | bool     | Disables output of the message text. Default is ``false``.                              |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| disable_fields      | bool     | Disables output of all fields. Default is ``false``.                                    |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| disables_stacktrace | bool     | Disables output of stack traces. Default is ``false``.                                  |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| timestamp_format    | string   | Format for timestamps. Default is `RFC3339 <https://www.rfc-editor.org/rfc/rfc3339>`__. |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+
+GELF log format configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Configure audit log in Boards
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++----------+----------+----------------------------------------------------------+
+| **Key**  | **Type** | **Description**                                          |
++----------+----------+----------------------------------------------------------+
+| hostname | string   | Outputs a custom hostname in log records.                |
+|          |          | If omitted, hostname is taken from the operating system. |
++----------+----------+----------------------------------------------------------+
 
-The `Boards configuration file <https://github.com/mattermost/focalboard/blob/main/config.json>`_ ``config.json`` is used to configure logging.
+Configure audit logging in Mattermost Boards
+--------------------------------------------
 
-``logging_cfg_file`` is used to specify the path to a file containing the logging configuration in JSON format.
-
-``logging_cfg_json`` is used to provide logging configuration directly as embedded JSON. Typically this is overridden using the corresponding environment variable ``FOCALBOARD_LOGGING_CFG_JSON``.
-
-Both configuration methods can be used, but care must be taken to avoid multiple log targets writing to the same file.
+The `Boards configuration file <https://github.com/mattermost/focalboard/blob/main/config.json>`_ ``config.json`` is used to configure logging. There are two settings which control logging:
```

Comment:
This is only for FocalBoard personal server and desktop edition.  Does not apply to Boards within the suite.

Reply:
This whole Boards section doesn't apply to suite.  I suggest it be removed.

Reply:
@wuwinson - Looping you in on Doug's suggestion. Which route would you prefer the Audit Logging documentation take with regards to Boards? Specifying that the content applies _only) to the personal server and desktop edition, OR removing this section of the docs entirely?

Reply:
Agreed with @wiggin77 - lets remove it so the docs don't confuse our users.